app/docker/containers/components/ContainersDatatable/ContainersDatatableActions.tsx

```diff
@@ -0,0 +1,295 @@
+import { useRouter } from '@uirouter/react';
+
+import * as notifications from '@/portainer/services/notifications';
+import { useAuthorizations, Authorized } from '@/portainer/hooks/useUser';
+import { Link } from '@/portainer/components/Link';
+import { confirmContainerDeletion } from '@/portainer/services/modal.service/prompt';
+import { setPortainerAgentTargetHeader } from '@/portainer/services/http-request.helper';
+import type { ContainerId, DockerContainer } from '@/docker/containers/types';
+import {
+  killContainer,
+  pauseContainer,
+  removeContainer,
+  restartContainer,
+  resumeContainer,
+  startContainer,
+  stopContainer,
+} from '@/docker/containers/containers.service';
+import type { EndpointId } from '@/portainer/endpoints/types';
+
+type ContainerServiceAction = (
+  endpointId: EndpointId,
+  containerId: ContainerId
+) => Promise<void>;
+
+interface Props {
+  selectedItems: DockerContainer[];
+  isAddActionVisible: boolean;
+  endpointId: EndpointId;
+}
+
+export function ContainersDatatableActions({
+  selectedItems,
+  isAddActionVisible,
+  endpointId,
+}: Props) {
+  const selectedItemCount = selectedItems.length;
+  const hasPausedItemsSelected = selectedItems.some(
+    (item) => item.Status === 'paused'
+  );
+  const hasStoppedItemsSelected = selectedItems.some((item) =>
+    ['stopped', 'created'].includes(item.Status)
+  );
+  const hasRunningItemsSelected = selectedItems.some((item) =>
+    ['running', 'healthy', 'unhealthy', 'starting'].includes(item.Status)
+  );
+
+  const isAuthorized = useAuthorizations([
+    'DockerContainerStart',
+    'DockerContainerStop',
+    'DockerContainerKill',
+    'DockerContainerRestart',
+    'DockerContainerPause',
+    'DockerContainerUnpause',
+    'DockerContainerDelete',
+    'DockerContainerCreate',
+  ]);
+
+  const router = useRouter();
+
+  if (!isAuthorized) {
+    return null;
+  }
+
+  return (
+    <div className="actionBar">
+      <div className="btn-group space-right" role="group">
+        <Authorized authorizations="DockerContainerStart">
+          <button
```

Comment:
and shouldn't this be a Button ?

Reply:
it should be used like this
`<Button
            color='success'
            size='small'
            onClick={() => onStartClick(selectedItems)}
            disabled={selectedItemCount === 0 || !hasStoppedItemsSelected}
          >`